connector_easypost/tests/__init__.py

```diff
@@ -3,11 +3,11 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from . import test_related_action
-from . import test_synchronization
-
-from . import test_res_partner
+ 
 from . import test_address
-from . import test_stock_picking_dispatch_rate
-from . import test_stock_picking
-from . import test_product_packaging
+from . import test_easypost_backend
+from . import test_res_partner
 from . import test_shipping_label
+from . import test_stock_picking
+from . import test_stock_picking_rate
+from . import test_stock_quant_package
```

Comment:
All review items have been addressed, @lasley. Coveralls are failing because somehow coverage from my last commit to this one dropped 0.1%? Overall, it's still +0.8% *shrug*.